source/_patterns/02-organisms/components/highlight~two-items.json

```diff
@@ -0,0 +1,36 @@
+{
+  "heading": {
+    "heading": "Highlights"
+  },
+  "items": [
+    {
+      "title": "[Do not use this base pattern directly: see the following variants of this pattern for usage]",
+      "url": "#link-to-article",
+      "summary": "Summary",
+      "meta": {
+        "text": "#"
+      },
+      "image": {
+        "fallback": {
+          "defaultPath": "https://unsplash.it/250/140/",
+          "altText": "Meaningful alt text please."
+        }
+      }
+    },
+    {
+      "title": "[Do not use this base pattern directly: see the following variants of this pattern for usage]",
```

Comment:
You should use actual examples.

Reply:
Real data is added in all examples for one, two and full items